__init__.py

```diff
@@ -0,0 +1,10 @@
+"""
```

Comment:
Do you know why this is necessary or what it does? Didn't need this before.. is it Windows-specific? Just wondering if it could cause any problems.